docs/customization.md

```diff
@@ -239,12 +239,38 @@ directly in the source of the theme and recompile it.
 
 ### Environment setup
 
-First, clone the repository:
+First, clone the repository for the edition you want to work on. Note
+that you need to be a member of [Insiders] to access its repository.
```

Comment:
Have we clarified in this document that there are two editions? Maybe we should clarify it with an "if", because it sounds a little too exclusive. Something like "If you want to clone the Insiders repository, you need to become a sponsor first to gain access.